contents/docs/new-to-posthog/understand-posthog.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Understanding PostHog
+---
+
+import { Admonition } from 'components/Docs/Admonition'
+
+PostHog is a lot of power, so to get the most from it, we should review how data is structured, grouped, and related.
+
+![PostHog data flow](https://res.cloudinary.com/dmukukwp6/image/upload/posthog_data_diagram_84bf3b121b.png)
```

Comment:
I think the diagram would actually be better at the bottom of the page – i.e. explain the concepts first, then bring in the diagram showing how they fit together.